packages/components/stories/GuidedTour.js

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import { I18nextProvider, translate } from 'react-i18next';
+import i18n, { LanguageSwitcher } from './config/i18n';
+
+import { GuidedTour } from '../src/index';
+import I18N_DOMAIN_COMPONENTS from '../src/constants';
+
+class ImportDemo extends React.Component {
+	constructor() {
+		super();
+		this.state = {
+			loading: false,
+			finish: false,
+		};
+		this.onClick = this.onClick.bind(this);
+	}
+
+	onClick() {
+		this.props.beforeLoading();
+		this.setState({ loading: true }, () => {
+			setTimeout(() => {
+				this.setState({ loading: false, finish: true }, () => {
+					this.props.afterLoading();
+					setTimeout(() => {
+						this.props.afterFinish();
+					}, 500);
+				});
+			}, 3000);
+		});
+	}
+
+	render() {
+		if (this.state.loading) {
+			return (
+				<div>
+					<strong>Loading ⏳</strong>
+					<br />
+					<small>Note that controls are disabled</small>
+				</div>
+			);
+		} else if (this.state.finish) {
+			return <span>Finish ✅</span>;
+		}
+		return (
+			<button className={'btn btn-info'} onClick={this.onClick}>
+				Simulate import
+			</button>
+		);
+	}
+}
+
+class GuidedTourContainer extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			isOpen: true,
+			controls: true,
+		};
+		this.closeTour = this.closeTour.bind(this);
+		this.showControls = this.showControls.bind(this);
+		this.hideControls = this.hideControls.bind(this);
+	}
+
+	closeTour() {
+		this.showControls();
+		this.setState({ isOpen: false });
+	}
+
+	showControls() {
+		this.setState({ controls: true });
+	}
+
+	hideControls() {
+		this.setState({ controls: false });
+	}
+
+	render() {
+		const { controls, isOpen } = this.state;
+		return (
+			<GuidedTour
+				steps={this.props.getSteps({
+					showControls: this.showControls,
+					hideControls: this.hideControls,
+					t: this.props.t,
+				})}
+				onRequestClose={this.closeTour}
+				isOpen={isOpen}
+				showCloseButton={controls}
+				showButtons={controls}
+				disableKeyboardNavigation={!controls}
+				disableAllInteractions={!controls}
+			/>
+		);
+	}
+}
+
+const TranslatedGuidedTourContainer = translate(I18N_DOMAIN_COMPONENTS)(GuidedTourContainer);
+
+// @see https://github.com/elrumordelaluz/reactour#steps
+function getSteps({ hideControls, showControls, t }) {
+	return [
+		{
+			selector: '[data-tour="language-switcher"]',
+			content: {
+				header: 'Hello world',
+				body: () => (
+					<div
+						dangerouslySetInnerHTML={{
```

Comment:
I guess because of the br in the translation. IMO we should avoid markup in translations anyway, and instead create 2 translations. a br is in 99% of cases two paragraphs. 

But I don't know if the reality will accept this opinion 😄

Reply:
Indeed
> I guess because of the br in the translation.

I can but how to deal with an unordered list? It will be a mess... And we have this kind of need.